src/System.Text.Json/src/System/Text/Json/JsonException.cs

```diff
@@ -7,8 +7,7 @@
 namespace System.Text.Json
 {
     /// <summary>
-    /// Defines a custom exception object that is thrown when invalid JSON text is encountered, when the defined maximum depth is passed,
-    /// or the JSON text is not compatible with the type of a property on an object.
+    /// Defines the base class for exceptions thrown from the System.Text.Json namespace.
```

Comment:
Specifying that it is only thrown from this namespace seems odd to me. @mairaw, do you have any recommendations?